src/app/App.tsx

```diff
@@ -1,9 +1,18 @@
 import { useState } from 'react';
+import { Container } from '@mui/material';
 
 import { Header } from 'ui';
+import { MainPage } from './mainPage/MainPage';
 
 export const App = () => {
   const [isLoggedIn, setIsLoggedIn] = useState(false);
 
-  return <Header isLoggedIn={isLoggedIn} />;
+  return (
+    <>
+      <Header isLoggedIn={isLoggedIn} />
+      <Container maxWidth="md">
+        <MainPage />
```

Comment:
Wiesz co, raczej się nie używa `MainPage`. To będzie `Dashboard`, prawda?
Właściwie to zależy od konwencji, ale jakoś ten dashboard będzie lepszy